scripts/docker/dev/nginx/default.conf

```diff
@@ -8,6 +8,9 @@ server {
     proxy_read_timeout          600;
     send_timeout                600;
 
+    # Docker's internal resolver
+    resolver 127.0.0.11;
```

Comment:
Will it be consistent across all docker installations for all its versions?

Reply:
It looks like it's been there for at least 5 years https://github.com/moby/moby/blame/a77317882d010b884a9101c6ad0b2d7db141082f/libnetwork/sandbox.go#L136